src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -157,19 +168,37 @@ int halide_hexagon_remote_set_performance(
         return -1;
     }
 
-    request.type = HAP_power_set_mips_bw;
-    request.mips_bw.set_mips        = set_mips;
-    request.mips_bw.mipsPerThread   = mipsPerThread;
-    request.mips_bw.mipsTotal       = mipsTotal;
-    request.mips_bw.set_bus_bw      = set_bus_bw;
-    request.mips_bw.bwBytePerSec    = ((uint64_t) bwMegabytesPerSec) << 20;
-    request.mips_bw.busbwUsagePercentage = busbwUsagePercentage;
-    request.mips_bw.set_latency     = set_latency;
-    request.mips_bw.latency         = latency;
-    retval = HAP_power_set(NULL, &request);
-    if (0 != retval) {
-        log_printf("HAP_power_set(HAP_power_set_mips_bw) failed (%d)\n", retval);
-        return -1;
+    if (use_dlopenbuf()) {
+        memset(&request, 0, sizeof(HAP_power_request_t));
+        request.type = HAP_power_set_DCVS_v2;
+        request.dcvs_v2.dcvs_enable = TRUE;
+        request.dcvs_v2.dcvs_option = HAP_power_dcvs_v2_payload::HAP_DCVS_V2_POWER_SAVER_MODE;
```

Comment:
Do we need to be able to control this? It seems like it might have a big impact on performance?

Reply:
We haven't observed any performance regression for hexagon_benchmark with HAP_DCVS_V2_POWER_SAVER_MODE, although there are other modes too (including PERF mode, which might penalize in terms of power). How do we combine the two - dcvs_option and target_corner? Or should we just leave it without specifying dcvs_option? Note that HAP_DCVS_V2_POWER_SAVER_MODE is the default option for all the clients participating in dcvs, and hence specified so.